src/modules/main/Main.tsx

```diff
@@ -29,10 +34,18 @@ const Main = ({ match }: RouteChildrenProps) => {
 
       <main>
         <Switch>
+          <Route
+            exact
+            path={`${match.path}/admin`}
+            render={() => <Redirect to={`${match.path}/admin/${AdminTabCategory.TECHNOLOGIES}`} />}
+          />
+
           <Route exact path={`${match.path}/dashboard`} render={() => <div>dashboard</div>} />
 
           <Route exact path={`${match.path}/projects`} render={() => <div>projects</div>} />
 
+          <Route path={`${match.path}/admin/:category`} component={AdminPanel} />
```

Comment:
AdminPanel name should be renamed to Admin - this it does not increase the readability and understanding of what it is. A similar situation to make a component and name it ComponentNameComponent